Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1908,43 +1908,48 @@ namespace PlayerCommands
 	bool CheckSolarDistances(uint client, uint systemID, Vector pos)
 	{
 		// Other POB Check
-		for (const auto& base : player_bases)
-		{
-			if (base.second->basetype != "legacy" 
-				|| base.second->invulnerable 
-				|| !base.second->logic
-				|| base.second->system != systemID
-				|| (base.second->position.x == pos.x
-					&& base.second->position.y == pos.y
-					&& base.second->position.z == pos.z))
-				continue;
-
-			float distance = HkDistance3D(pos, base.second->position);
-			if (distance < minOtherPOBDistance)
+		if (minOtherPOBDistance > 0)
+		{
+			for (const auto& base : player_bases)
 			{
-				if (client)
-				{
-					PrintUserCmdText(client, L"Player base %ls is too close! Minimum distance: %um", base.second->basename.c_str(), static_cast<uint>(minOtherPOBDistance));
-				}
-				else
+				if (base.second->basetype != "legacy"

```

Comment:
When you have an if like this you should document the control flow in a comment. Ideally avoid checks like this because this is very hard to understand. Especially with brackets being used near the end with things like position.

Reply:
And while I know that bases were legacy class, it's worth documenting what that actually means to anyone who is going to look at this later.

Reply:
Good point. Also made me realize the check as it was would allow players to plonk POBs next to admin placed objects.